src/features/comments.ts

```diff
@@ -0,0 +1,80 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+
+type CommentsState = {
+  comments: Comment[];
+  loaded: boolean;
+  error: boolean;
+};
+
+const initialState: CommentsState = {
+  comments: [],
+  loaded: false,
+  error: false,
+};
+
+export const addComment = createAsyncThunk(
+  'comments/add',
+  (comment: Omit<Comment, 'id'>) => {
+    return createComment(comment);
+  },
+);
+
+export const removeComment = createAsyncThunk(
+  'comments/remove',
+  (commentId: number) => {
+    return deleteComment(commentId);
+  },
+);
```

Comment:
The `removeComment` async thunk does not handle the fulfilled or rejected states. Consider adding cases for these states in the `extraReducers` to manage the UI state appropriately when a comment is removed.